microsoft-edge/web-platform/local-network-access.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Controlling Local Network Access in Microsoft Edge
+description: Local Network Access in Microsoft Edge enables users to grant or deny sites' access to your local network.  # key words before col 158
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/18/2025
+---
+# Controlling Local Network Access in Microsoft Edge
+<!-- todo: improve "description:" above, after clarify intro -->
+
+Use the Local Network Access feature to grant or deny sites' access to your<!-- todo: who is "your"?  the (end-)user's? --> local network.
+<!-- todo:
+As an end-user, you can use the Local Network Access feature to grant or deny a website access to your local network.
+
+state the audience; clarify the audience and scenario.  Admins?  Devs?  End-users?
+what admin must configure the policies, for this to work?
```

Comment:
what admin must configure the policies, for this to work, so that the end-user is able to control L.N.A.?
* `LocalNetworkAccessAllowedForUrls`
* `LocalNetworkAccessBlockedForUrls`